src/backend/core/factories.py

```diff
@@ -124,6 +124,7 @@ class UserFactory(factory.django.DjangoModelFactory):
 
     class Meta:
         model = models.User
+        django_get_or_create = ("email",)
```

Comment:
nit : did it not get or create before ? 🤔

Reply:
I am not sure to understand, I aligned the `UserFactory` with the `IdentityFactory`

Reply:
I thought it already "get_or_created" the objet, but I imagine it only "created" it without being able to retrieve existing object.